source/_integrations/google_generative_ai_conversation.markdown

```diff
@@ -28,12 +30,18 @@ The Google Generative AI API key is used to authenticate requests to the Google
 
 {% include integrations/option_flow.md %}
 {% configuration_basic %}
-Prompt Template:
-  description: The starting text for the AI language model to generate new text from. This text can include information about your Home Assistant instance, devices, and areas and is written using [Home Assistant Templating](/docs/configuration/templating/).
+Model:
```

Comment:
Duplicate.

Reply:
Removing the duplicate in https://github.com/home-assistant/home-assistant.io/pull/32857